src/accounts.js

```diff
@@ -227,17 +227,48 @@ async function createAccount(password, salt, ownerPublicKey, orePayerAccountName
   };
 }
 
+// returns a list of actions to link to an app permission 
+// every { contract, action } input pair is linked to the app permission
+async function getLinkActions(links, permission, authAccountName, authPermission){
```

Comment:
getLinkActions sounds like you're retrieving something. Since this function actually generates a structure to send to a function, a name like composeLinkActions is a better name.